tensorflow/compiler/tf2tensorrt/utils/trt_int8_calibrator.h

```diff
@@ -69,9 +69,9 @@ struct TRTInt8Calibrator : public nvinfer1::IInt8EntropyCalibrator {
   void setDone();
 
   // If not null, calibration is skipped.
-  const void* readCalibrationCache(std::size_t& length) override;
+  const void* readCalibrationCache(std::size_t& length) noexcept override;
 
-  void writeCalibrationCache(const void* ptr, std::size_t length) override;
+  void writeCalibrationCache(const void* ptr, std::size_t length) noexcept override;
```

Comment:
please also add noexcept to the implementation.

Reply:
Added